kong/dao/schemas/apis.lua

```diff
@@ -83,8 +83,13 @@ local function check_uri(uri)
     return false, "invalid"
 
   elseif not match(uri, "^/[%w%.%-%_~%/%%]*$") then
-    -- Check if characters are in RFC 3986 unreserved list, and % for percent encoding
-    return false, "must only contain alphanumeric and '., -, _, ~, /, %' characters"
+    -- URI contains characters outside of the reserved list of
+    -- RFC 3986: the value will be interpreted as a regex;
+    -- but is it a valid one?
+    local _, _, err = ngx.re.find("", uri)
```

Comment:
Maybe `"aj"`? Although to the best of my knowledge, I don't see why they would result in an error and not the default flags. I'm not feeling strongly about it, but if we decide to, then the cors plugin's schema, which is using the same trick, should maybe do so as well. I don't think we should used `o` though.